cmd/sandbox-ce/cmd/web.go

```diff
@@ -70,10 +70,62 @@ func OpenUI(timeout time.Duration) error {
 		ch <- openUI()
 	}()
 
+	if timeout > 5*time.Second {
+		stopSpinner := startSpinner("Initializing Source{d}...")
```

Comment:
I've always seen source{d} brand all lowercase.
```suggestion
		stopSpinner := startSpinner("Initializing source{d}...")
```